dask/array/fft.py

```diff
@@ -86,19 +64,64 @@ def _ihfft_out_chunks(a, n, axis):
     return chunks
 
 
-fft = _fft_wrap(npfft.fft, np.complex_, _fft_out_chunks)
+out_chunk_fns = {'fft': _fft_out_chunks,
+                 'ifft': _fft_out_chunks,
+                 'rfft': _rfft_out_chunks,
+                 'irfft': _irfft_out_chunks,
+                 'hfft': _hfft_out_chunks,
+                 'ihfft': _ihfft_out_chunks}
```

Comment:
So I'm not sure it is a good idea to make `out_chunk_fns` as part of the public API. Particularly when what it is mapping are functions that are not part of the public API.